discord/guild.py

```diff
@@ -138,6 +138,7 @@ class Guild(Hashable):
         - ``MORE_EMOJI``: Guild is allowed to have more than 50 custom emoji.
         - ``DISCOVERABLE``: Guild shows up in Server Discovery.
         - ``FEATURABLE``: Guild is able to be featured in Server Discovery.
+        - ``COMMUNITY``: Guild is a community server.
```

Comment:
Since line 143 uses the term "public guild" instead of "public server", should this one also say "community guild" instead?